didact/src/components/login/Login.jsx

```diff
@@ -4,75 +4,79 @@ import * as Yup from 'yup';
-        "& *": {
-            width: '80%',
-            border: 'solid #9B9B9B 1.2px',
-            backgroundColor: '#F7F7F7',
-            height:'2.5em'
-        }
-    },
-    password: {
-        width: "70%",
-        "& *": {
-            width: '80%',
-            border: 'solid #9B9B9B 1.2px',
-            backgroundColor: '#F7F7F7',
-            height:'2.5em'
-        }
-    },
-    loginButton: {
-        width: '25%',
-        height:'2.7em',
-        textAlign: 'center',
-        backgroundColor: '#5A5A5A',
-        color: 'white',
-        border: 'transparent',
-        borderRadius: '5%'
-    },
-    buttonContainer: {
-        paddingTop: '2%'
-    },
-    footer: {
-        display: 'flex',
-        justifyContent: 'space-between',
-        width: '50%',
-        height: '2.7em',
-        margin: '0 auto',
-        paddingTop: '2%',
-        "& *":{
-            height:'100%',
-            backgroundColor: '#5A5A5A',
-            color: 'white',
-            border: 'transparent',
-            borderRadius: '5%',
-            width: '33%',
-            textDecoration: 'none'
-        }
-    }
+    // header: {
+    //     width: '50%',
+    //     margin: '0 auto',
+    //     '& *': {
+    //         textAlign: 'center'
+    //     } //testtest
+    // },
+    // container: {
+    //     display: 'flex',
+    //     flexDirection: 'column',
+    //     alignItems: 'center',
+    //     justifyContent: 'center',
+    //     justifyContent: 'space-evenly',
+    //     width: '45%',
+    //     height: '10em',
+    //     border: 'solid #9B9B9B 1.2px',
+    //     borderRadius: '1%',
+    //     margin: '0 auto'
+    // },
+    // email: {
+    //     width: "70%",
+    //     "& *": {
+    //         width: '80%',
+    //         border: 'solid #9B9B9B 1.2px',
+    //         backgroundColor: '#F7F7F7',
+    //         height:'2.5em'
+    //     }
+    // },
+    // password: {
+    //     width: "70%",
+    //     "& *": {
+    //         width: '80%',
+    //         border: 'solid #9B9B9B 1.2px',
+    //         backgroundColor: '#F7F7F7',
+    //         height:'2.5em'
+    //     }
+    // },
+    // loginButton: {
+    //     width: '25%',
+    //     height:'2.7em',
+    //     textAlign: 'center',
+    //     backgroundColor: '#5A5A5A',
+    //     color: 'white',
+    //     border: 'transparent',
+    //     borderRadius: '5%'
+    // },
+    // buttonContainer: {
+    //     paddingTop: '2%'
+    // },
+    // footer: {
+    //     display: 'flex',
+    //     justifyContent: 'space-between',
+    //     width: '50%',
+    //     height: '2.7em',
+    //     margin: '0 auto',
+    //     paddingTop: '2%',
+    //     "& *":{
+    //         height:'100%',
+    //         backgroundColor: '#5A5A5A',
+    //         color: 'white',
+    //         border: 'transparent',
+    //         borderRadius: '5%',
+    //         width: '33%',
+    //         textDecoration: 'none'
+    //     }
+    // }
```

Comment:
All these comments, learn to trust the commit process :)
j/k, I don't know how to do all that stuff either. We should have like, 2 weeks on just git.